examples/selective2d/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+NUM_GPUS=8
+
+# 1. run dimension solver
+python3 ./dim_solver/main.py --num-gpu $NUM_GPUS --opt-ar1
+
+# 2. process output file
+# output file name: solver.out
+# output data: pp_group_size tp_group_size size_microbatch i_stage n_chunks
+
+pp_group_size=$(cat ./solver.out | awk '{print $1}')
```

Comment:
Hard-code the name to be a fixed name is not a good idea. You can add timestamp so that we will not have duplicate file name for different script runs. Or we removed the file in the end.. You can decide it yourself.